server/start.go

```diff
@@ -59,6 +59,16 @@ const (
 	FlagStateSyncSnapshotInterval   = "state-sync.snapshot-interval"
 	FlagStateSyncSnapshotKeepRecent = "state-sync.snapshot-keep-recent"
 
+	// api-related flags
+	flagAPIEnable             = "api.enable"
+	flagAPISwagger            = "api.swagger"
+	flagAPIAddress            = "api.address"
+	flagAPIMaxOpenConnections = "api.max-open-connections"
+	flagRPCReadTimeout        = "api.rpc-read-timeout"
+	flagRPCWriteTimeout       = "api.rpc-write-timeout"
+	flagRPCMaxBodyBytes       = "api.rpc-max-body-bytes"
+	flagAPIEnableUnsafeCORS   = "api.enabled-unsafe-cors"
```

Comment:
@faddat pointed out, rightfully so, that all flags should really be public and exposed 👍 

let's start doing that with this set.